rules/coding-style/src/Rector/Encapsed/EncapsedStringsToSprintfRector.php

```diff
@@ -123,8 +123,16 @@ private function collectExpr(Expr $expr): void
     private function createSprintfFuncCallOrConcat(string $string, array $argumentVariables): Node
     {
         // special case for variable with PHP_EOL
-        if (count($argumentVariables) === 2 && $argumentVariables[1] instanceof ConstFetch && (string) $argumentVariables[1]->name === 'PHP_EOL') {
-            return new Concat($argumentVariables[0], $argumentVariables[1]);
+        if ($string === '%s%s' && count($argumentVariables) === 2 &&
+            count(
+                array_filter(
+                    $argumentVariables,
+                    fn($argumentVar): bool => $argumentVar instanceof ConstFetch && $argumentVar->name.'' === 'PHP_EOL'
```

Comment:
argument->name . '' should can be (string) argument->name

Reply:
@samsonasik thanks, guessing you're also building against >7.3 as it doesn't like the return type on the closure